### ISSUE TITLE
Bug in 'reverse-proxy with sub-path' configuration

### ISSUE THREAD

**Issue by arladmin** [nocodb/nocodb#428]
**Please enter the following details**

Node: **v12.22.1**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **pg**
ProjectOnRootDB: **true**
RootDB: **pg**
PackageVersion: **0.11.9**

**Steps To Reproduce**
1. Set up a reverse proxy, e.g. nginx
2. Set up a location directive in nginx, where a subfolder is forward to nocodb:
```
server {
    server_name sub.domain.tld;
    listen 80;
    location /db/ {
            proxy_set_header Host $host;
            proxy_set_header X-Forwarded-Host $host:$server_port;
            proxy_set_header X-Forwarded-Proto https;
            proxy_set_header X-Real-IP $remote_addr;
            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
            proxy_pass http://localhost:8080/;
            chunked_transfer_encoding off;
            proxy_buffering off;
            proxy_cache off;
    }
    location / {
      // another app
    }
} 
```
In the docker-compose, set up the environment variables:
```
  nocodb:
    container_name: nocodb
    depends_on:
      root_db:
        condition: service_healthy
    image: nocodb/nocodb:latest
    ports:
      - "8080:8080"
      - "127.0.0.1:8081:8081"
      - "127.0.0.1:8082:8082"
      - "127.0.0.1:8083:8083"
    restart: always
    environment:
      NC_DB: "db connection string" 
      NC_PUBLIC_URL: "http://sub.domain.tld/db"
      NC_BACKEND_URL: "http://sub.domain.tld/db"
      NC_DISABLE_TELE: "true"
```
4. The app starts behaving erroneously. Some links in the UI don't work correctly while some options might be missing.
- Clicking on a project sends an invalid request: ```http://sub.domain.tld/nc/test1_XnM0/projectApiInfo``` 
    As ```/db/``` is not prefixed before ```/nc/```, nginx considers this request to be invalid and actually sends it to the other app.
- After opening a project, in the top-left pop-up menu, the link for Swagger Docs is missing.
![image](https://user-images.githubusercontent.com/64059154/128353999-62c6c8e5-6f70-4b12-8630-16de1f5f4ccb.png)

This link is available when the nocodb app is accessed directly via the server's IP address (bypassing nginx):
![image](https://user-images.githubusercontent.com/64059154/128354062-f5375d54-e617-42f3-939a-2bcf8b463545.png)

**Expected behaviour**
The app should load and operate smoothly and all routes should be prefixed with: "http://sub.domain.tld/db/"

**Comment by pranavxc** [comment]
Fixed in `0.11.10`

**Comment by pranavxc** [comment]
Fixed swagger issue in `0.11.11`
